dashboard/cron.py

```diff
@@ -298,13 +305,16 @@ def update_with_bq_access(self):
             # Because we're pulling all the data down into one query we need to manipulate it a little bit
             # Make a copy of the access dataframe
             resource_df = resource_access_df.copy(deep=True)
-            # Drop out the columns user and access time from resource data frame
-            resource_df.drop(["user_id", "access_time"], axis=1, inplace=True)
+            # Drop out the columns user course_id and access time from resource data frame
+            resource_df = resource_df.drop(["user_id", "course_id", "access_time"], axis=1)
             # Drop out the duplicates
-            resource_df.drop_duplicates(["resource_id", "course_id"], inplace=True)
+            resource_df = resource_df.drop_duplicates(["resource_id",])
+
+            # Set a dual index for upsert
+            resource_df = resource_df.set_index(["resource_id", "course_id"])
 
-            # Drop out the columns resource_type, course_id, name from the resource_access
-            resource_access_df.drop(["resource_type","name", "course_id"], axis=1, inplace=True)
+            # Drop out the columns resource_type, name from the resource_access
+            resource_access_df = resource_access_df.drop(["resource_type","name"], axis=1)
```

Comment:
```suggestion
            # Drop out the columns resource_type and name from the resource_access
            resource_access_df = resource_access_df.drop(["resource_type", "name"], axis=1)
```

Reply:
Similar note about the comment here; the code basically says exactly what the comment does.